index.md

```diff
@@ -5,9 +5,13 @@ layout: default
 <div class="lead pretty-links">
 StarCon is a two-day software engineering conference held at the **University of Waterloo**. Join us for two days of ten minute lighting talks about technology, computer science, programming, and the joy of learning about them!
 
+## Who's speaking? Can I apply to speak?
+
+Check out our [invited speakers](speakers/#keynotes)! Talk proposal details will be posted in September.
```

Comment:
I guess I see them all as invited speakers, and keynotes and invited lightning speakers as separate

but I don't know if other people would get that, heh - so I'll explicitly say keynotes thank ^^